### ISSUE TITLE
Remove bind from globals list in documentation

### ISSUE THREAD

**Issue by laurentlb** [bazelbuild/bazel#11308]
https://docs.bazel.build/versions/3.1.0/skylark/lib/globals.html#bind

bind is not a valid function in BUILD/bzl files.
We should move it or remove it.

**Comment by laurentlb** [comment]
The same applies to `workspace`, `repository_rule`.

If it's complicated to change the documentation generator, we can add a comment in the doc to clarify.
Put a link to the workspace documentation.

**Comment by aiuto** [comment]
So, is this a doc generation problem or an external deps problem?
I'm leaving it without a team to see if someone wants to pick it up during the doc fixit.

**Comment by keertk** [comment]
Docs moved [here](https://bazel.build/rules/lib/globals#bind)

**Comment by aiuto** [comment]
I don't think we want external help on this. We need a full restructuring of the API documentation. That is going to take a lead from the team and a tech writer.

**Comment by Wyverald** [comment]
This was done last year. The globals have been split into different pages, one for each environment.
